### DISCUSSION TITLE
[doc] Where to add documentation?

### DISCUSSION THREAD

**Discussion by matifraga** [pmd/pmd#3033] [Q&A] [answered]
**Description**
Hi, we, as Team Raptor, would like to add a documentation page (yeah, we are awesome) over our work to support CPD on Antlr-based languages, so that any future contributor could just follow a simple set of steps in order to enable support for new languages (and make all of our lives a little bit happier). 

Where would be the best place for it to be?

**Comment by jsotuyod** [comment, marked as answer]
If you are writing docs intended for developers, and particularly for CPD support, I think you are looking for this page: https://pmd.github.io/pmd-6.11.0/pmd_devdocs_major_adding_new_cpd_language.html

That page is generated from this file https://github.com/pmd/pmd/blob/master/docs/pages/pmd/devdocs/major_contributions/adding_new_cpd_language.md

It's quite poor as it stands so feel free to revise it at will.
